scripts/install-helpers/baremetal-coco/intel-dcap/pccs.yaml.in

```diff
@@ -0,0 +1,123 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pccs-config
+  namespace: intel-dcap
+data:
+  default.json: |
+    {
+      "HTTPS_PORT" : 8042,
+      "hosts" : "0.0.0.0",
+      "uri": "https://api.trustedservices.intel.com/sgx/certification/v4/",
+      "ApiKey" : "${PCCS_API_KEY}",
+      "proxy" : "${HTTP_PROXY}",
+      "RefreshSchedule": "0 0 1 * * *",
+      "UserTokenHash" : "${PCCS_USER_TOKEN_HASH}",
+      "AdminTokenHash" : "${PCCS_ADMIN_TOKEN_HASH}",
+      "CachingFillMode" : "LAZY",
+      "OPENSSL_FIPS_MODE" : false,
+      "LogLevel" : "debug",
+      "DB_CONFIG" : "sqlite",
+      "sqlite" : {
+        "database" : "${PCCS_DB_NAME}",
+        "username" : "${PCCS_DB_USERNAME}",
+        "password" : "${PCCS_DB_PASSWORD}",
+        "options" : {
+          "host": "localhost",
+          "dialect": "sqlite",
+          "pool": {
+            "max": 5,
+            "min": 0,
+            "acquire": 30000,
+            "idle": 10000
+          },
+          "define": {
+            "freezeTableName": true
+          },
+          "logging" : true,
```

Comment:
PCCS default:
```suggestion
          "logging" : false,
```

Reply:
I'd prefer keeping at least some log enabled, as otherwise we'll make our lives quite hard when it comes to debugging it.  I'll tone down the log level as you suggested from `debug` to `info`.

Thanks!